twitchobserver/twitchobserver.py

```diff
@@ -190,6 +190,14 @@ def send_whisper(self, user, message):
 
         self.send_message("/w {} {}".format(user, message), None)
 
+    def change_color(self, color):
+        """Changes the color of your name in chat.
+
+        :param color: The new name color. Can be a hex value like #000000 or one of the following: Blue, Coral, DodgerBlue, SpringGreen, YellowGreen, Green, OrangeRed, Red, GoldenRod, HotPink, CadetBlue, SeaGreen, Chocolate, BlueViolet or Firebrick
```

Comment:
It might be nice to add the named colors to the twitchobserver package.
Something like:
```python
observer.change_color(twitchobserver.colors.CORAL)
```

Thoughts?